sdk/core/azure-core/tests/test_streaming.py

```diff
@@ -43,6 +44,17 @@ def test_decompress_plain_no_header(http_request):
     decoded = content.decode('utf-8')
     assert decoded == "test"
 
+@pytest.mark.parametrize("http_request", HTTP_REQUESTS)
```

Comment:
love that you're parameterizing this test with our two different kinds of http requests!

for more background information, we have an older `HttpRequest` class and a newer one. Eventually, we want to move to the newer one, but we want to make sure our tests run on both of them, so thank you for making this test run with both request classes!

Reply:
Thank you!! I appreciate learning the background on why both kinds of http requests are needed here. 😊 You are welcome!